### ISSUE TITLE
Cant dim

### ISSUE THREAD

**Issue by tolleburker** [robertklep/name.klep.sonoff#37]
Hi how can i custom changed the dim topic, because it is posting in the Dimmer topic
ive been using an flashed LSC light to tasmota.
and when i look into the mqtt explorer i can changed it when i pubblised in the Dimmer topic.
but the app is using the dim topic

**Comment by robertklep** [comment]
As far as I remember, dimming isn't supported by the Sonoff app at all.

**Comment by tolleburker** [comment]
but its possible 😁

**Comment by robertklep** [comment]
Not without adding the feature to the app, though. And I stopped developing the app :(

**Comment by tolleburker** [comment]
but is it possible that i can change an topic ?

**Comment by robertklep** [comment]
No, it's not possible. I don't really understand how the app is publishing dim messages to MQTT when I don't see how that's actually supported 🤔

**Comment by tolleburker** [comment]
its publissing in a homey topic 
<img width="530" alt="image" src="https://user-images.githubusercontent.com/59849333/72558868-dbff7c80-38a3-11ea-9290-8ded49eac0aa.png">

**Comment by robertklep** [comment]
Ah, that's not done by the Sonoff app but by the MQTT Hub app.
